src/Modules/AuthModule/components/Register/Register.tsx

```diff
@@ -1,7 +1,287 @@
+import React, { useContext, useState } from "react";
+import { useForm } from "react-hook-form";
+import img1 from "../../../../assets/images/PMS 3.png";
+import Avatar from "../../../../assets/images/Ellipse 1.png";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { toast } from "react-toastify";
+import { AuthContext } from "../../../../context/AuthContext";
 
 export default function Register() {
+ let {baseUrl}= useContext(AuthContext)
+  const {
+    handleSubmit,
+    register,
+    watch,
+    formState: { errors },
+  } = useForm();
+  const Navigate = useNavigate();
+  const [selectedImage, setSelectedImage] = useState("");
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+  const togglePasswordVisibility = (field: any) => {
+    if (field === "password") {
+      setShowPassword(!showPassword);
+    } else if (field === "confirmPassword") {
+      setShowConfirmPassword(!showConfirmPassword);
+    }
+  };
+
+  function AppendToFormData(data: any) {
+    const formData = new FormData();
+    formData.append("userName", data.userName);
+    formData.append("profileImage", data.profileImage[0]);
+    formData.append("email", data.email);
+    formData.append("phoneNumber", data.phoneNumber);
+    formData.append("country", data.country);
+    formData.append("password", data.password);
+    formData.append("confirmPassword", data.confirmPassword);
+    return formData;
+  }
+
+  function changingImage(event: any) {
+    setSelectedImage(URL.createObjectURL(event.target.files[0]));
+  }
+
+  async function onSubmit(data: any) {
+    let SubmitData = AppendToFormData(data);
+    try {
+      const res: any = await axios.post(
+        `${baseUrl}/Users/Register`,
+        SubmitData
+      );
+      toast.success(res?.data?.message);
```

Comment:
We should provide a fallback message, because sometimes there is no message coming from the response.